website/docs/docs/build/entities.md

```diff
@@ -150,3 +150,96 @@ entities:
     type: foreign # This can be any entity type key. 
     expr: date_key || '|' || brand_code # Defines the expression for linking fields to form the surrogate key.
 ```
+
+## Examples
+
+As mentioned, entites serve as our join keys, using the unique entity name. Therefore, we can join a single `unique` key to multiple `foreign` keys.
+
+Consider `date_categories` table with the following columns:
+
+```sql
+date_id (primary key)
+date_day (unique key)
+fiscal_year_name
+```
+
+And a `orders` table with the following columns:
+```sql
+order_id (primary key)
+ordered_at
+delivered_at
+order_total
+```
+
+How might we define our Semantic Layer yml, so we can query `order_total` by `ordered_at` `fiscal_year_name` and `delivered_at` `fiscal_year_name`?
```

Comment:
```suggestion
How might we define our Semantic Layer YAML so that we can query `order_total` by `ordered_at` `fiscal_year_name`, and `delivered_at` `fiscal_year_name`?
```